commons/src/main/scala/com/lynbrookrobotics/potassium/commons/drivetrain/unicycle/UnicycleCoreControllers.scala

```diff
@@ -98,7 +98,8 @@ trait UnicycleCoreControllers {
   def forwardPositionControl(targetAbsolute: Stream[Length])
                             (implicit hardware: DrivetrainHardware,
                              props: Signal[DrivetrainProperties]): (Stream[UnicycleSignal], Stream[Length]) = {
-    val error: Stream[Length] = targetAbsolute.minus(hardware.forwardPosition)
+    // drop one to get in sync with pid
+    val error: Stream[Length] = targetAbsolute.minus(hardware.forwardPosition).drop(1)
 
```

Comment:
What are the `drop(1)`s for?

Reply:
PID controllers do not output a value for the first sensor input (because of derivative/integral). This changes the error to be in-sync with the output of the PID stream.